var/spack/repos/builtin/packages/intel-oneapi-itac/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import platform
+
+from spack.package import *
+
+
+@IntelOneApiPackage.update_description
+class IntelOneapiItac(IntelOneApiPackage):
+    """The Intel® Trace Analyzer and Collector profiles and analyzes MPI applications to help
+    focus your optimization efforts.
+
+    Find temporal dependencies and bottlenecks in your code.
+    Check the correctness of your application.
+    Locate potential programming errors, buffer overlaps, and deadlocks.
+    Visualize and understand parallel application behavior.
+    Evaluate profiling statistics and load balancing.
+    Analyze performance of subroutines or code blocks.
+    Learn about communication patterns, parameters, and performance data.
+    Identify communication hot spots.
+    Decrease time to solution and increase application efficiency.
+
+    """
+
+    homepage = "https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/trace-analyzer.html"
+
+    if platform.system() == "Linux":
```

Comment:
Curious.  Why not add the `version()` directive then a `conflicts()` if the platform is not `Linux`?

Could use something along the following (or in a loop):

```
    conflicts("platform=cray", msg="Cray is not supported")
    conflicts("platform=darwin", msg="macOS is not supported")
    conflicts("platform=windows", msg="Windows is not supported")
```

Reply:
I held close to the related `intel-oneapi-*` packages. I think we should either change them all at once or none. And, thinking of it, adding support for linux seems more robust to me than removing support for all others. @rscohn2 what are your thoughts?

Reply:
It would be good to move the logic here: https://github.com/spack/spack/blob/a0c7209dc1480b16531fcb992642986f4796bc28/lib/spack/spack/build_systems/oneapi.py#L22
Is it possible to do something like?
```
conflicts("platform!=linux", msg="Platform is not supported")
```
That would be a separate PR, and leave this PR consistent with all the others.

Reply:
```
conflicts("platform!=linux", msg="Platform is not supported")
```
doesn't work. Having:
```
    conflicts("platform=cray", msg="Cray is not supported")
    conflicts("platform=darwin", msg="macOS is not supported")
    conflicts("platform=windows", msg="Windows is not supported")
```

in the base class would be better than logic in every derived class as it is now. It would be even better if the logic was that linux was supported instead of everything else not supported. @alalazo do you have any suggestions?

Reply:
@tldahlgren: Are you ok with merging this as-is?

I am willing to change it to something like:
```
    conflicts("platform=cray", msg="Cray is not supported")
    conflicts("platform=darwin", msg="macOS is not supported")
    conflicts("platform=windows", msg="Windows is not supported")
```

in a separate commit that changes all packages so it is in the base class. I am still hoping for a conflicts that says ``not linux`` instead of enumerating all platforms, but that is not a big deal.